spec/index.bs

```diff
@@ -25,7 +25,32 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
 Introduction {#intro}
 =====================
 
-This specification rocks.
+*This section is non-normative.*
+
+[Graphics Processing Units](https://en.wikipedia.org/wiki/Graphics_processing_unit), or GPU for short,
+have been essential in enabling rich rendering and computational applications in personal computing.
+WebGPU is an API that exposed the capabilities of GPU hardware for the web.
```

Comment:
Very nitpicky, but shouldn't it be "*exposes*", in present tense? 😄

Reply:
nicely spotted! this is a mistake